css_02/css_02.css

```diff
@@ -0,0 +1,122 @@
+html *
+{
+   font-family: Arial, Helvetica, sans-serif !important;
+}
+
+.header 
+{
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    width: 91%;
+    height: 40px;
+}
+
+.signupButton 
+{
+    width: 100px;
+}
+
+.loginButton 
+{
+    width: 60px;
+}
+
+.greenButtons 
+{
+    height: 30px;
+    border-radius: 12px;
+    border: 1.5px solid rgb(169, 202, 56);
+    background-color: #97bf0e;
+    color: #fff;
+    margin: 6px 6px;
+}
+
+.whiteButtons
+{
+    height: 30px;
+    border-radius: 12px;
+    border: 1.5px solid rgb(169, 202, 56);
+    background-color: #fff;
+    color: rgb(151, 191, 14);
+    margin: 6px 6px;
+}
+
+.navigationBar
+{
+    text-transform: uppercase;
+    width: 25%;
+    margin: 20px 20px 0 50px;
+}
+
+.navigationBar ul
+{
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+.navigationBar li
+{
+    border-bottom: 1px solid rgb(114, 114, 114);
+    padding: 10px 15px;
+}
+
+.container
+{
+    display: flex;
+}
+
+main
+{
+    width: 60%;
+}
+
+main h1
+{
```

Comment:
Here you can use element selector, instead of element element selector.  Both are doing the same job in this case.